File: Leetcode-Problems/Excel Sheet Column Title.cpp
class Solution
{
public:
    string convertToTitle(int columnNumber)
    {
        string ans = "";
        while (columnNumber > 0)
        {
            int dig = (columnNumber - 1) % 26;
            ans += (dig + 'A');
            columnNumber = (columnNumber - 1) / 26;
        }
        reverse(ans.begin(), ans.end());
        return ans;
    }
};

File: Leetcode-Problems/Excel Sheet Column Title.java
class Solution {
    public String convertToTitle(int columnNumber) {
        StringBuilder ans = new StringBuilder();
        while (columnNumber > 0) {
            int rem = (columnNumber - 1) % 26;
            ans.append((char) (rem + 'A'));
            columnNumber = (columnNumber - 1) / 26;
        }

        return ans.reverse().toString();
    }
}